backend/battle/tests/test_view.py

```diff
@@ -1,15 +1,104 @@
-from django.test import TestCase
 from model_bakery import baker
-from battle.models import PokemonTeam
+from common.utils.tests import TestCaseUtils
```

Comment:
I didn't see this class, is it in another PR?

Reply:
This already exists on the boilerplate :)